ext/cairo/rb_cairo_font_face.c

```diff
@@ -39,6 +39,11 @@ static ID cr_id_at_need_clusters;
 static ID cr_id_at_need_cluster_flags;
 #endif
 
+#if CAIRO_HAS_FT_FONT
+static FT_Library library;
```

Comment:
No, the font is not the same as the library. We only need one library per instance here as each library can load an arbitrary number of fonts. See [`FT_Library`](http://www.freetype.org/freetype2/docs/reference/ft2-base_interface.html#FT_Library)